setup.cfg

```diff
@@ -0,0 +1,34 @@
+[metadata]
+license_file = LICENSE.rst
+
+[bdist_wheel]
+universal = 1
+
+[tool:pytest]
+testpaths = tests
+filterwarnings =
+    error
+
+[flake8]
+# B = bugbear
+# E = pycodestyle errors
+# F = flake8 pyflakes
+# W = pycodestyle warnings
+# B9 = bugbear opinions,
+# ISC = implicit str concat
+select = B, E, F, W, B9, ISC
+ignore =
+    # line length, handled by black
+    B950
+    E501
+    # bare except, handled by bugbear B001
+    E722
+    # binary operation line break, different opinion from black
+    W503
+# up to 88 allowed by bugbear B950
+max-line-length = 80
+per-file-ignores =
```

Comment:
this where you can add exemptions to flake8 rules, the filter ones i'd like to address longer term but it might involve a breaking change